GVRf/Framework/framework/src/main/java/org/gearvrf/io/GVRGearCursorController.java

```diff
@@ -304,41 +298,22 @@ public void setScene(GVRScene scene) {
         showControllerModel(mShowControllerModel);
     }
 
-    @Override
-    public void invalidate() {
-        if (!initialized) {
-            //do nothing
-            return;
-        }
-        thread.sendInvalidate();
-    }
-
     public void onDrawFrame()
     {
         boolean wasConnected = mConnected;
 
         mConnected = (mControllerReader != null) && mControllerReader.isConnected();
         if (!mConnected)
         {
-            if (initialized)
+            if (wasConnected)
```

Comment:
I don't think after merging with master the changes to this method do what is intended. There is no gaze cursor controller for me in any app.